bigdata-client/src/main/java/com/bigdata/rdf/sail/remote/AbstractBigdataRemoteQuery.java

```diff
@@ -27,8 +27,8 @@ protected void configureConnectOptions(IPreparedQuery q) {
 		}
 		q.addRequestParam(RemoteRepositoryDecls.INCLUDE_INFERRED,
 				Boolean.toString(includeInferred));
-		if (maxQueryTime > 0) {
-			q.addRequestParam(RemoteRepositoryDecls.MAX_QUERY_TIME_MILLIS, Long.toString(1000L*maxQueryTime));
+		if (getMaxExecutionTime() > 0) {
```

Comment:
Was maxQueryTime a field in the base class? Make sure you know the units involved.  Open RDF historically used seconds.  But it might be milliseconds with 2.8 and the new method name.  If so, chase how that value gets set from the serpents and make sure the units are consistent.

Reply:
Yes, AbstractQuery had protected int maxQueryTime = 0;, which got removed later. getMaxQueryTime and setMaxQueryTime existed too, and worked on that same variable, and now they do setMaxExecutionTime() & getMaxExecutionTime(). 
So I am pretty confident the units remained the same and there's no conversion between these two functions.

Reply:
http://archive.rdf4j.org/javadoc/sesame-2.8.4/org/openrdf/query/Operation.html#setMaxExecutionTime-int-

I checked.  Yes, it is still seconds.